### PR TITLE
WIP #401 update documentation with user feedback

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
@PavelBal This is the branch that I'd like you to start from to finish what is required in #401 
I think these suggestions are good and it should not take you too much time to implement most of them. If something is too complicated, let's skip

File: vignettes/sensitivity-analysis.Rmd [hunk 1]
```diff
@@ -39,11 +39,11 @@ sa <- SensitivityAnalysis$new(simulation = sim, parameterPaths = c("Aciclovir|Li
                                                                       "Organism|Age"))
 print(sa)
 
-# Show which parameters will be variated
+# Show which parameters will be varied
```
Comment (msevestre) [inline vignettes/sensitivity-analysis.Rmd:42]: FYI @PavelBal Variated does not exist in English

File: R/ospsuite-env.R [hunk 2]
```diff
@@ -43,3 +43,22 @@ ospsuiteEnv$sensitivityAnalysisConfig$totalSensitivityThreshold <- 0.9
...
+#'
+#' @param settingName String name of the setting
+#'
+#' @return Value of the setting stored in ospsuiteEnv. If the setting does not exist, an error is thrown.
+#' @export
+#'
+#' @examples
+#' getOSPSuiteSetting("packageVersion")
+#' getOSPSuiteSetting("sensitivityAnalysisConfig")$totalSensitivityThreshold
+getOSPSuiteSetting <- function(settingName){
```
Comment (PavelBal) [inline R/ospsuite-env.R:58]: @msevestre Added this method as proposed in https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/386 to remove mentioning of ospsuitEnv in the class documentation.

File: R/sensitivity-analysis-run-options.R [hunk 3]
```diff
@@ -10,8 +10,8 @@ SensitivityAnalysisRunOptions <- R6::R6Class(
   public = list(
     #' @description
     #' Initialize a new instance of the class
-    #' @param numberOfCores Number of cores to use for the simulation. Default value is `ospsuiteEnv$numberOfCores`
-    #' @param showProgress Should a progress information be displayed. Default value is `ospsuiteEnv$showProgress`
+    #' @param numberOfCores Number of cores to use for the simulation. Default value is \code{getOSPSuiteSetting("numberOfCores")}
```
Comment (PavelBal) [inline R/sensitivity-analysis-run-options.R:13]: Replaces the direct access to (not exposed) ospsuiteEnv.

File: vignettes/create-run-population.Rmd [hunk 4]
```diff
@@ -123,4 +124,20 @@ resultsValues <- resultsData$data$`Organism|PeripheralVenousBlood|Aciclovir|Plas
 plot(resultsTime, resultsValues, type = "l")
 ```
 
+To get the results for a specific individual or a set of individuals, the argument `individualIds` of the method `getOutputValues` can be specified:
+
```
Comment (PavelBal) [inline vignettes/create-run-population.Rmd:128]: Addressing point 4 from https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/401

File: vignettes/load-get.Rmd [hunk 5]
```diff
@@ -130,11 +130,11 @@ sim <- loadSimulation(simFilePath)
...
-
+ 
 # Calling simTree would list all entities within the simulation
 # Accessing the parameter "Organism|Weight"
 simTree$Organism$Weight
-
+# 
 # Getting all entities located under "Organism|Liver|Periportal|Intracellular"
-simTree$Organism$Liver$Periportal$Intracellular
+entitiesList <- simTree$Organism$Liver$Periportal$Intracellular
```
Comment (PavelBal) [inline vignettes/load-get.Rmd:139]: Somehow I could not build the vignette without assigning the value to a variable.

File: vignettes/run-simulation.Rmd [hunk 6]
```diff
@@ -62,6 +65,21 @@ plot(resultsTime, resultsValues, type = "l")
 
 The results can be stored in and imported from a *.csv file with the methods `exportResultsToCSV` and `importResultsFromCSV`.
 
+```{r SimResultsExport}
```
Comment (PavelBal) [inline vignettes/run-simulation.Rmd:68]: Addresssing point 3 from https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/401

File: vignettes/sensitivity-analysis.Rmd [hunk 7]
```diff
@@ -20,72 +20,79 @@ The models built with OSPS depend on a lot of input parameters which are based o
 In brief, the values of the chosen parameters are changed by a certain percentage and the impact of these changes on PK parameters of model outputs is assessed.
 
 ## Performing a sensitivity analysis
-The first step of performing a sensitivity analysis is creating an object of the class `SensitivityAnalysis`. At this step, the user can define which parameters should be considered for the sensitivity analysis, in which range the values are variated, and how may steps are performed in one direction (plus and minus). If no parameters are specified, all constant parameters of the simulation will be variated.
+The first step of performing a sensitivity analysis is creating an object of the class `SensitivityAnalysis`. At this step, the user can define which parameters should be considered for the sensitivity analysis, in which range the values are varied, and how may steps are performed in one direction (plus and minus). If no parameters are specified, all constant and suitable for sensitivity calculations parameters of the simulation will be variated. The list of such parameters paths for a simulation can be accessed with the method `potentialVariableParameterPathsFor(simulation)`.
```
Comment (PavelBal) [inline vignettes/sensitivity-analysis.Rmd:23]: As was requested by @Yuri05 in https://github.com/Open-Systems-Pharmacology/OSPSuite-R/pull/390

File: vignettes/sensitivity-analysis.Rmd [hunk 8]
```diff
@@ -95,13 +102,14 @@ sensitivites <- saResult$allPKParameterSensitivitiesFor(pkParameterName = "AUC_i
...
 The value `-1` for the sensitivity of "AUC_inf" of the output `Organism|PeripheralVenousBlood|Aciclovir|Plasma (Peripheral Venous Blood)` for the parameter `Organism-Kidney-Volume` means that increasing `Organism-Kidney-Volume` by 10% will decrease "AUC_inf" by 10%.
+Note that the list of sensitivities is ordered from largest to smallest with respect to magnitude.
 
 ## Import and export of sensitivity analysis results
 Sensitivity analysis calculated in R can be exported to a *.csv file, which can be loaded in another instance.
 
 ```{r SAExport}
 # Load and run the simulation
-simFilePath <- file.path(getwd(), "..", "tests", "data", "Aciclovir.pkml", fsep = .Platform$file.sep)
+simFilePath <- file.path(getwd(), "..", "tests", "data", "simple.pkml", fsep = .Platform$file.sep)
```
Comment (PavelBal) [inline vignettes/sensitivity-analysis.Rmd:112]: This should speed up building of vignettes (https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/392)

File: vignettes/sensitivity-analysis.Rmd [hunk 9]
```diff
@@ -20,72 +20,79 @@ The models built with OSPS depend on a lot of input parameters which are based o
...
+The method `allPKParameterSensitivitiesFor` returns a list of `PKParameterSensitivity` objects. `PKParameterSensitivity` describes the sensitivity (field `$value`) of a PK-Parameter (`$pkParameterName`) for the output `$outputPath` calculated for the varied parameter `$parameterPath`. The argument `totalSensitivityThreshold` of the method `allPKParameterSensitivitiesFor` is used to filter out the most impactful parameters. A threshold of 0.9 means that only parameters participating to a total of 90 percent of the sensitivity would be returned. A value of 1 would return the sensitivity for all parameters. If no value is provided, a default value is used.
```
Comment (PavelBal) [inline vignettes/sensitivity-analysis.Rmd:82]: As was requested by @Yuri05 in #390

File: vignettes/set-values.Rmd [hunk 10]
```diff
@@ -39,7 +39,7 @@ ageParam$isConstant
 ```
 If the value is not constant, it is described by one of the formula types:
 
-  **Distributed**: The value of the parameter is given by a distribution. In the simulation, the behavior equals that of constant values.
+  **Distributed**: Distributed parameters describe a variation around a constant value or between two numerical limits. In the simulation, the behavior equals that of constant values. See [OSPS documentation](https://docs.open-systems-pharmacology.org/working-with-mobi/mobi-documentation/model-building-components#working-with-constant-and-distributed-parameters) for more information.
```
Comment (PavelBal) [inline vignettes/set-values.Rmd:42]: Point 2 from https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/401

**Comment by PavelBal** [comment]
@msevestre please review.

File: R/ospsuite-env.R (hunk 2 above)
Comment (msevestre) [reply R/ospsuite-env.R:58]: ok good enough. We do not allow to write the values. We might want to do that for another PR

**Comment by msevestre** [comment]
@PavelBal So here, only the gif suggested is missing correct? Looks good to me. We can leave the gif for now :)

**Pull request merged**